src/ShallowWaterIntegrators.jl

```diff
@@ -0,0 +1,141 @@
+function assemble_rhs_vector(A, B, dc)
+  # short cut routine to assemble a domain contribution into a vector
+  # A:  test function
+  # B:  trial function
+  # dc: domain contribution
+  data  = Gridap.FESpaces.collect_cell_vector(B, dc)
+  assem = SparseMatrixAssembler(A, B)
+  rhs   = assemble_vector(assem, data)
+end
+
+function shallow_water_explicit(model, order, Ω, dΩ, dω, qₖ, wₖ, f, g, h₁, u₁, hₘ, uₘ, RTMM, L2MM, dt, leap_frog, τ, P, Q, U, V, R, S)
```

Comment:
why do you need `qₖ, wₖ` ?

Reply:
why do you need `Ω` ?

Reply:
why do you need `order`?

Reply:
can we choose a more precise name for the `shallow_water_explicit` function? For example, one that, apart from referring to the method at hand, tells the reader about what does the function actually does. E.g., does it move the time forward?